rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+DRIVERS=="cmpc_accel_v4", \
```

Comment:
It doesn't work with this rule removed. But admittedly I'm running a slightly old systemd with 60-sensor rules and hwdb stuff backported, I may have missed something?

If this rule is not here, in which other execution path will the sensor: hwdb values be queried and acted upon?

There is one other rule in this file but that one is specifically for iio i2c/usb devices.